### ISSUE TITLE
prefix window title with number of unread messages so you can see it in the taskbar of the window manager e.g.

### ISSUE THREAD

**Issue by darix** [dino/dino#144]
similar to what github and so do.

**Comment by zinid** [comment]
Any chance this issue will even be considered? This is a stopper issue for me, because I hate popups. I also think this will not over-complicate the application's design or make it geeky.

**Comment by RalfJung** [comment]
I agree this would be very useful. I often miss Gnome's notifications (or have them muted entirely since they interrupt my concentration). Other chat applications put an unread message counter in the window title or some indication into the icon so a quick glance at my taskbar and browser tab bar immediately tells me "I have some unread messages in Signal, Zulip, Matrix, and Mattermost; let's check those out". But dino doesn't do anything like that. I regularly completely miss messages because of that and then my contacts ask why I don't reply.

Maybe if Gnome's notification system worked better (e.g., grouping by application, like on Android), just sending a notification would be enough, but they way it works I usually have around 100 notifications at the end of a day and then just discard them all since it's not possible to go through all of them.

I hope https://github.com/dino/dino/pull/1336 can be merged :)
